src/ucs/datastruct/mpool.c

```diff
@@ -58,14 +58,15 @@ static void ucs_mpool_chunk_leak_check(ucs_mpool_t *mp, ucs_mpool_chunk_t *chunk
 
 ucs_status_t ucs_mpool_init(ucs_mpool_t *mp, size_t priv_size,
                             size_t elem_size, size_t align_offset, size_t alignment,
-                            unsigned elems_per_chunk, unsigned max_elems,
+                            unsigned elems_per_chunk, double grow_factor, unsigned max_elems,
```

Comment:
the number of mpool_init parameters becomes very large.
maybe add new struct ucs_mpool_params_t to contain all input parameters for creating the memory pool?

Reply:
added ucs_mpool_params_t